core/domain/fs_domain.py

```diff
@@ -98,6 +125,16 @@ def _get_file_metadata(self, filepath, version):
         """Return the desired file metadata.
 
         Returns None if the file does not exist.
+
+        Args:
+            filepath: str. The path to the relevant file within the exploration.
+            version: int. The version number of the file whose metadata is to be
+            returned.
+
+        Returns:
+            FileMetadataModel or None: The model object representing the file
+            metadata with the given exploration_id, filepath and version or None
```

Comment:
Here and below: Add comma or semicolon after "version", to separate it from the second part of the sentence.